go.mod

```diff
@@ -3,33 +3,35 @@ module github.com/GoogleCloudPlatform/alloydb-auth-proxy
 go 1.24
 
 require (
-	cloud.google.com/go/alloydbconn v1.14.1
+	cloud.google.com/go/alloydbconn v1.15.0
```

Comment:
LMK if we should just merge in https://github.com/GoogleCloudPlatform/alloydb-auth-proxy/pull/760 instead of updating the Go Connector in this PR. Our checks didn't like it when I had only updated the Go Connector, hence the other updates :(

Reply:
This is fine.